include/re_dbg.h

```diff
@@ -49,47 +49,87 @@ enum {
  *
  * Print warning message
  */
+
+/**
+ * @def DEBUG_WARNING_ID(...)
+ *
+ * Print warning message with identifier
+ */
+
 #if (DEBUG_LEVEL >= 4)
-#define DEBUG_WARNING(...) \
+#define DEBUG_WARNING(...)                                                    \
 	dbg_printf(DBG_WARNING, DEBUG_MODULE ": " __VA_ARGS__)
+#define DEBUG_WARNING_ID(...)                                                 \
+	dbg_printf_id(debug_id, DBG_WARNING, DEBUG_MODULE ": " __VA_ARGS__)
```

Comment:
is it better to pass "debug_id" via the macro parameters, instead of assume the name ?

Reply:
It follows the same technique used for DEBUG_LEVEL and DEBUG_MODULE.
The compiler errors if `debug_id` is not defined. I think it's much easier for rewriting existing DEBUG code since you have only define `debug_id` and add `_ID`. And this needs less reformatting new lines, that was the idea behind it.

Reply:
Should we consider uppercase: `DEBUG_ID` ?